website/pages/blog/judging-open-source-by-github-stars.mdx

```diff
@@ -0,0 +1,254 @@
+---
+title: Judging Open Source by Github Stars
+tags: [open-source]
+authors: yassin
+date: 2023-05-09
+description: Exposing the dark side of GitHub and open source projects
+image: /blog-assets/judging-open-source-by-github-stars/cover.png
+thumbnail: /blog-assets/judging-open-source-by-github-stars/thumbnail.png
+---
+
+The best and most obvious way to judge an open-source project is to look at the code but this
+can be kind of tedious and sometimes you don't like what you see there, so an alternative that we
+have all naturally developed on our own or have been advised to, is to see how many people have
+starred a project, and then pick the one with the most stars.
+
+"For example, React.js has 207K stars compared to Angular's measly 88K stars, so we can conclude
+that React.js is a better framework" — [Ben Awad](https://youtu.be/MT6M_sqAuZo?t=243)
+
+While the above is a joke (not that React isn't a better frontend framework), it is not a good idea
+to judge the quality of an open-source project by how many stars it has.
+
+How many times have you gazed upon an open source project, and wondered fabulously about the
+correlation between the number of stars garnered and the actual value of the project?
+
+If it's zero, then ok, you're a better person than me.
+
+But if not, then allow me to introduce you to my recent experiments and research. It's likely that
+this will be your first encounter with this bizarre market.
+
+## Buying GitHub Stars
+
+Yep! That's a thing. Just as purchasing Instagram likes is a genuine phenomenon, buying GitHub stars
+is equally tangible.
+
+![Buy instagram likes poster](/blog-assets/judging-open-source-by-github-stars/buy-instagram-likes.png)
+
+---
+
+But, I won't just tell you that buying GitHub stars is real, and move on, that would be so
+disappointing, right?
+
+So I did it! I have tried 2 platforms, I won't say any names or links, as I would never advocate for
+such a thing.
```

Comment:
shouldn't we really?
I mean, its not necessarily advocating but more exposing...

Reply:
I guess yeah, I wasn't sure if it was gonna do more harm than good 🤷‍♂️